lib/get-format.js

```diff
@@ -18,7 +18,9 @@ const extensionFormatMap = {
   '.cjs': 'commonjs',
   '.js': 'module',
   '.json': 'json',
-  '.mjs': 'module'
+  '.mjs': 'module',
+  // Taken from https://github.com/nodejs/node/blob/595ce9dac63cdd32d893c3319328a752719644d2/lib/internal/modules/esm/formats.js#L28-L30
+  '.wasm': 'wasm'
```

Comment:
I've added the wasm module as a builtin format since it looks like that's the intention from as shown in the link provided.